templates/web/pages/org_profile/org_profile.tt.html

```diff
@@ -14,6 +14,41 @@
 	<p>&rarr; <a href="[% link %]">[% lang("official_site") %]</a></p>
 [% END %]
 
+[% IF org_members.size %]
+<h2>[% lang("organization_members") %]</h2>
+	<p>[% lang("number_of_members") %][% sep %]: [% org_members.size %]</p>
+	<div style="position:relative">
+	<table id="org_main_table">
+		<thead id="org_table_header">
+		<tr>
+			<th>[% lang("serial_no") %]</th>
+			<th>[% lang("member_userid") %]</th>
+			<th>[% lang("name") %]</th>
+			<th>[% lang("email") %]</th>
+			<th>[% lang("select_lang") %]</th>
+			<th>[% lang("select_country") %]</th>
+
+		</tr>
+	</thead>
+		[% SET count = 1 %]
+		[% FOREACH users IN org_members %]
+		<tbody id="org_table_body">
```

Comment:
the tbody element should be outside the FOREACH loop, otherwise you will have a table with more than 1 tbody.